DfE.FindInformationAcademiesTrusts/Services/Export/ExportBuilder.cs

```diff
@@ -0,0 +1,83 @@
+﻿using DfE.FindInformationAcademiesTrusts.Pages;
+using DfE.FindInformationAcademiesTrusts.Services.Trust;
+
+namespace DfE.FindInformationAcademiesTrusts.Services.Export
+{
+    using ClosedXML.Excel;
```

Comment:
[Microsoft recommends](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#place-the-using-directives-outside-the-namespace-declaration) keeping using statements outside the namespace